### PR TITLE
helper utility to fix up the IP address in LND config (as well as modified LND conf)

### PR DESCRIPTION
**Pull request by nolim1t**
To work in conjunction with PR https://github.com/lncm/pi-factory/pull/179

### CODE REVIEW

**Comment by nolim1t** [comment]
Objections or thoughts?

This is more an edge case but it supports TOR only nodes

**Comment by nolim1t** [comment]
I've amended this pull request to support TOR nodes @AnotherDroog 

You still have to uncomment stuff for it to work right now

**Comment by AnotherDroog** [comment]
Awesome, going to test this

File: usr/local/sbin/update-lnd-tor-ipaddress.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IP=`ip route get 1 | awk '{print $NF;exit}'`
+
+# Look for existance of .ipaddress.txt
+if [[ ! -f /home/lncm/.ipaddress.txt ]]; then
```
Comment (AnotherDroog) [inline usr/local/sbin/update-lnd-tor-ipaddress.sh]: [[ isn't valid ash syntax

File: usr/local/sbin/update-lnd-tor-ipaddress.sh [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# This script is a helper script that rewrites the host IP to the new IP Address (used for TOR only nodes).
```
Comment (AnotherDroog) [inline usr/local/sbin/update-lnd-tor-ipaddress.sh]: No `#!/bin/sh` ?

File: usr/local/sbin/update-lnd-tor-ipaddress.sh (hunk 1 above)
Comment (nolim1t) [reply usr/local/sbin/update-lnd-tor-ipaddress.sh]: It works in ASH for me.

File: usr/local/sbin/update-lnd-tor-ipaddress.sh (hunk 2 above)
Comment (nolim1t) [reply usr/local/sbin/update-lnd-tor-ipaddress.sh]: done

File: usr/local/sbin/update-lnd-tor-ipaddress.sh (hunk 1 above)
Comment (nolim1t) [reply usr/local/sbin/update-lnd-tor-ipaddress.sh]: I added #!/bin/sh anyway

**Comment by nolim1t** [comment]
This is just a stopgap measure for now and will be retired as soon as we get tor as a container in docker

**Comment by AnotherDroog** [comment]
Works, merging

**Pull request merged**